src/resources/postcss/tickets-commerce/admin/orders/single.pcss

```diff
@@ -1,3 +1,59 @@
+.tec-tickets-commerce-vertical-form label {
+	display: block;
+	margin-bottom: 2px;
```

Comment:
Is there a mock for this?
2px is not one of our spacers, so it's suspect. (smallest is 4px)
Same with 6px (below - should be 4px or 8px)
16px is - and it's used several times below. (i'm not going to tag each one, that gets messyI

Reply:
PS: found the project mock